tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "include": ["src", "cjsmin", "cdv", "tests"],
   "compilerOptions": {
+    "module": "commonjs",
```

Comment:
fyi @bennyrubanov this should be the key fix. The issue is that jest compatibility is different than for the modules themselves and I don't believe i validated that for the heatmap PR

Reply:
ok good to know. thx for highlighting and finding the fix